examples/api/lib/widgets/text_magnifier/text_magnifier.0.dart

```diff
@@ -0,0 +1,110 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/foundation.dart';
+import 'package:flutter/material.dart';
+
+void main() => runApp(const MyApp());
+
+class MyApp extends StatelessWidget {
+  const MyApp({super.key});
+
+  static const String textFieldText = 'Custom Magnifier!';
+  static const Size loupeSize = Size(200, 200);
+
+  @override
+  Widget build(BuildContext context) {
+    return MaterialApp(
+      home: Scaffold(
+        body: Padding(
+          padding: const EdgeInsets.symmetric(horizontal: 48.0),
+          child: Center(
+            child: TextField(
+              // Don't worry about the key. Just for testing!
+              key: GlobalKey(),
+              // Create a custom magnifier configuration that
+              // this `TextField` will use to build a magnifier with.
+              magnifierConfiguration: TextMagnifierConfiguration(
+                magnifierBuilder: (BuildContext context,
```

Comment:
nit: not easiest to read. Maybe move the function gesture to the same line?

Reply:
I voided the params that are unneeded. Maybe this makes it read better?